# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/eh_frame.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dietlibc-0.29/dyn_start.c.orig	2005-07-07 15:09:10.000000000 +0200
+++ dietlibc-0.29/dyn_start.c	2005-07-07 15:09:31.000000000 +0200
@@ -24,7 +24,7 @@
 
 __attribute__((section(".eh_frame")))
 __attribute_used
-char __EH_FRAME_BEGIN__[] = { };
+static const char __EH_FRAME_BEGIN__[] = { };
 
 
 static void __do_global_dtors_aux(void)
--- dietlibc-0.29/dyn_stop.c.orig	2005-07-07 15:14:23.000000000 +0200
+++ dietlibc-0.29/dyn_stop.c	2005-07-07 15:17:58.000000000 +0200
@@ -17,9 +17,9 @@
 __attribute__((section(".eh_frame")))
 __attribute_used
 #if __WORDSIZE == 32
-char __FRAME_END__[4] = { 0, 0, 0, 0 };
+static const char __FRAME_END__[4] = { 0, 0, 0, 0 };
 #else
-char __FRAME_END__[8] = { 0, 0, 0, 0, 0, 0, 0, 0 };
+static const char __FRAME_END__[8] = { 0, 0, 0, 0, 0, 0, 0, 0 };
 #endif
 
 static void __do_global_ctors_aux(void)
